app/routers/search.py

```diff
@@ -0,0 +1,43 @@
+from app.database.database import get_db
+from app.dependencies import templates
+from app.internal.search import get_results_by_keywords
+from fastapi import APIRouter, Depends, Form, Request
+from fastapi.templating import Jinja2Templates
+from sqlalchemy.orm import Session
+
+
+router = APIRouter()
+templates = Jinja2Templates(directory="app/templates")
+
+
+@router.get("/search")
+def search(request: Request):
+    current_username = "Chuck Norris"
+    return templates.TemplateResponse("search.html", {
+        "request": request,
+        "username": current_username
+    })
+
+
+@router.post("/search")
+async def show_results(request: Request, keywords: str = Form(None), db: Session = Depends(get_db)):
+    current_username = "Chuck Norris"

```

Comment:
Could be nice to add a TODO comment here

Reply:
Will do!